lib/pages/general/SignUpInstructionsPage.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:grade_plus_plus/pages/AbstractPage.dart';
+import 'package:grade_plus_plus/pages/fragments/InstructionsStepDescription.dart';
+import 'package:grade_plus_plus/pages/fragments/DotsRow.dart';
+
+// Represents the page that contains the instructions on how to get your grades from sis
+class SignUpInstructionsPage extends AbstractPage {
+  SignUpInstructionsPage({Key key})
+      : super(
+          key: key,
+          appBarTitle: 'How to upload your grades',
+          appBarColorBg: Colors.blue,
+          appBarColorTxt: Colors.white,
+        );
+
+  _SignUpInstructionsPageState createState() => _SignUpInstructionsPageState();
+}
+
+class _SignUpInstructionsPageState extends PageState<SignUpInstructionsPage> {
+  int imgIndex = 0;
+
+  final List<String> imgList = [
+    'assets/signup_instructions1.png',
+    'assets/signup_instructions2.png',
+    'assets/signup_instructions3.png',
+    'assets/signup_instructions4.png',
+    'assets/signup_instructions5.png',
+    'assets/signup_instructions6.png',
+  ];
+
+  final List<String> descriptionsList = [
+    'Visit https://sis.auth.gr/old and login with your academic account.',
```

Comment:
This should be a link. Maybe if it works we can change it to https://sis.auth.gr/old/el/all_grades and reduce the number of pages.

Reply:
Commit with id 0db22d meets proposed change.

![image](https://user-images.githubusercontent.com/57541732/118804595-8ed35680-b8ad-11eb-9a1c-f4d938b4f7a3.png)